### ISSUE TITLE
[A11y]At 150% page zoom images present of left side of screen are not visible.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#7880]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1063665
Steps to Reproduce:
1. Hit the #URL:https://www.nuget.org/
2. Navigate to "Sign in" control and hit enter.
3. Navigate to "Sign in with Microsoft" control and hit enter.
4. Search for a key term (say “json”) from search box.
5. "Packages returned for json" screen gets opened.
6. Zoom the page to 150%.
7. Check if the images present on left side of the screen are visible. 

Actual:
When the page is zoomed to 150% and above upto 200% the images that are present on left side of the screen are not visible.

Same issue is Observed in "Chrome" also.

Expected:
At 150% page zoom all the content and images that are visible in 100% zoom must also be visible at 150% and above upto 200%.

Notes:
This is actually an issue with the 150% zoom shrinking the horizontal width of the page to the point that the bootstrap stuff kicks in and hides the column with the icon. If I recall, this was by design as we decided that the icon eats up too much space at that width. We should revisit this decision and see if we still agree that this is the correct behaviour.

**Comment by ryuyu** [comment]
I believe this is by design, and we will not be fixing this.
We hide the icon explicitly since too much of the horizontal real estate is consumed by the icon
